packages/perseus-editor/src/widgets/free-response-editor.tsx

```diff
@@ -108,6 +121,32 @@ class FreeResponseEditor extends React.Component<Props> {
                         }
                     />
                 </label>
+                <label
+                    className={css(styles.textOptionWithLabelContainer)}
+                    htmlFor="allow-unlimited-characters"
```

Comment:
Do you need the `id` and `htmlFor` if the label is wrapping the checkbox? If not, I'd advocate for removing them.

Reply:
Interesting. I added that to silence the [jsx-a11y/label-has-associated-control](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md#case-my-label-and-input-components-are-custom-components) ESLint issue.

There are a few places where this is ignored and those reference a TODO of [LEMS-2871](https://khanacademy.atlassian.net/browse/LEMS-2871). I added the same comment here.